docs/topics/mta-7-installing-web-console-on-openshift.adoc

```diff
@@ -118,6 +118,10 @@ The most commonly used CR settings are listed in this table:
 |`1Gi`
 |Size requested for the Keycloak database volume
 
+|`keycloak_sso_req_passwd_update`
```

Comment:
@nachandr - you said that `The keycloak_sso_req_passwd_update flag is missing in the documentation`, please could I ask you to provide the following values

```
|`keycloak_sso_req_passwd_update`
| <default value... presume Boolean false> 
| <description>
```
Thanks very much